.github/workflows/tests.yml

```diff
@@ -52,7 +53,7 @@ jobs:
             packages.microsoft.com:443
       - uses: actions/checkout@v4
       - run: sudo apt-get update; sudo apt-get install ${{ matrix.shell }}
-        if: matrix.shell == 'zsh' || matrix.shell == 'ksh'
+        if: matrix.shell == 'zsh' || matrix.shell == 'ksh' || matrix.shell == 'bash'
```

Comment:
why does bash need to be installed separately?

Reply:
Line 55 gets skipped otherwise making the build fail in line 59 when we try to run which ${{ matrix.shell }} 
https://github.com/nvm-sh/nvm/actions/runs/11578693670/job/32233129550

Reply:
right, but that would mean that bash isn't installed by default in the ubuntu runner image?

Reply:
I believe either that or maybe is not able to find it's path.. i can see if i can figure out what is happening..

Reply:
hmm, maybe it's that the `apt-get update` needs to happen unconditionally, but bash doesn't need to be installed - either way let's land this as-is and iterate.

Reply:
Seems like the slow tests are a little bit unstable, re-running a verification it actually cause the urchin ones to fail, looking into them right now if this could have been an issue with the node version..